instance/models/mixins/openedx_config.py

```diff
@@ -453,6 +462,15 @@ def _get_configuration_variables(self):
             template["EDXAPP_CMS_ENV_EXTRA"]["ADDL_INSTALLED_APPS"].append(hb_app)
             template["EDXAPP_LMS_ENV_EXTRA"]["ADDL_INSTALLED_APPS"].append(hb_app)
 
+        # See https://github.com/edx/cs_comments_service/pull/323
+        if self._is_openedx_release_in(['ironwood']):
+            template["forum_source_repo"] = "https://github.com/open-craft/cs_comments_service.git"
+            template["forum_version"] = "opencraft-release/ironwood.2"
+        if self._is_openedx_release_in(['juniper']):
+            template["forum_source_repo"] = "https://github.com/open-craft/cs_comments_service.git"
+            template["forum_version"] = "opencraft-release/juniper.2"
+            template["FORUM_VERSION"] = "opencraft-release/juniper.2"
```

Comment:
FYI @swalladge We have to specify both `forum_version` and `FORUM_VERSION` here, because [we set `"forum_version": self.openedx_release` in Ocim above](https://github.com/open-craft/opencraft/blob/7ff66c37fcd4dd52486f54772081a807281a9fa0/instance/models/mixins/openedx_config.py#L186-L187), and [edx:configuration juniper.2 uses `forum_version`](https://github.com/edx/configuration/blob/open-release/juniper.2/playbooks/roles/forum/tasks/deploy.yml#L45) still. [edx:configuration master drops `forum_version` and uses `FORUM_VERSION`](https://github.com/edx/configuration/blob/642db0e17837d758748982e7b95bcd399bb39bae/playbooks/roles/forum/tasks/deploy.yml#L45), so post-juniper, we can drop these lowercase variables.

To demonstrate, [this sandbox failed to provision](https://stage.manage.opencraft.com/instance/5530/edx-appserver/2116/), because it ended up with: 
```yaml
FORUM_VERSION: opencraft-release/juniper.2
forum_source_repo: https://github.com/open-craft/cs_comments_service.git
forum_version: open-release/juniper.2  # does not exist on this fork ^
```



the [configuration code for juniper is dumb](https://github.com/edx/configuration/blob/open-release/juniper.2/playbooks/roles/forum/defaults/main.yml#L92-L93) (it's fixed in [master]()):